### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by jsteemann**
This PR includes 2 small changes:
* fix typo in README.md
* fix typo in xxh3.h code comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ and passes all tests, ensuring reasonable quality levels.
 It also passes extended tests from [newer forks of SMHasher], featuring additional scenarios and conditions.
 
 Finally, xxHash provides its own [massive collision tester](https://github.com/Cyan4973/xxHash/tree/dev/tests/collisions),
-able to generate and compare billions of hash to test the limits of 64-bit hash algorithms.
+able to generate and compare billions of hashes to test the limits of 64-bit hash algorithms.
```
Comment (Cyan4973) [inline README.md:78]: :+1:

File: xxh3.h [hunk 2]
```diff
@@ -42,7 +42,7 @@
  * but it is still provided for compatibility with source code
  * which used to include it directly.
  *
- * Programs are now highly discourage to include xxh3.h.
+ * Programs are now highly discouraged to include xxh3.h.
```
Comment (Cyan4973) [inline xxh3.h:45]: :+1:

File: xxhsum.c [hunk 3]
```diff
@@ -658,7 +658,7 @@ static int XSUM_hashFile(const char* fileName,
         XSUM_setBinaryMode(stdin);
     } else {
         if (XSUM_isDirectory(fileName)) {
-            XSUM_log("xxhsum: %s: Is a directory \n", fileName);
+            XSUM_log("xxhsum: %s: Is a directory\n", fileName);
```
Comment (Cyan4973) [inline xxhsum.c]: These space characters are intentional.

They make the printed line more readable
when the printing overwrites a screen's section which is already filled with garbage characters.

I recommend removing this file's modifications from your PR.

**Review by Cyan4973 — CHANGES_REQUESTED** [review]
Final space characters on printed lines are intentional.
Please do not remove them.

**Comment by jsteemann** [comment]
Ok, changes to error messages reverted as suggested.

**Review by Cyan4973 — APPROVED** [review]

**Pull request merged**